211/3/restaurant/app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ReviewsController < BaseController
+  post '/reviews/create' do
```

Comment:
not really restful, use `post '/reviews'` or better `post '/restaurants/:id/reviews'` instead